frontend/src/main.tsx

```diff
@@ -1,10 +1,7 @@
-import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 import '@/index.css';
 import { Router } from '@/router';
 
 createRoot(document.getElementById('root')!).render(
-  <StrictMode>
```

Comment:
Don't remove strict mode, undo these changes and update the GamePage.tsx as:
Since the useEffect is running twice for the first render, remove it and create a function to handle the question change. When user enters first time, render a simple text that `Type /start to start the game`...
Add a condition, if user types `/start` then start the game.